File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/PaymentNotice.py
from typing import Optional, List
from base import CodeableConcept
from base import DomainResource
from base import Money
from base import Reference
from base import Identifier

class PaymentNotice(DomainResource):
    response: Optional[Reference] = None
    amount: Money
    request: Optional[Reference] = None
    payment: Reference
    recipient: Reference
    created: str
    payment_status: Optional[CodeableConcept] = None
    status: str
    payee: Optional[Reference] = None
    payment_date: Optional[str] = None
    identifier: Optional[List[Identifier]] = None
    provider: Optional[Reference] = None

File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/PaymentNotice.ts
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { Money } from "./Money";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type PaymentNotice = DomainResource & {
    response?: Reference;
    amount: Money;
    request?: Reference;
    payment: Reference;
    recipient: Reference;
    created: string;
    paymentStatus?: CodeableConcept;
    status: string;
    payee?: Reference;
    paymentDate?: string;
    identifier?: Identifier[];
    provider?: Reference;
};